<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceInformationOptionsDialog</class>
 <widget class="QDialog" name="DistanceInformationOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Distance Information Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="m_groupBox_distanceType">
     <property name="title">
      <string>Distance type:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QRadioButton" name="m_radioButton_centroid">
        <property name="text">
         <string>Centroid to centroid</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_radioButton_surface">
        <property name="text">
         <string>Surface to surface</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>m_radioButton_surface</zorder>
     <zorder>m_radioButton_centroid</zorder>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_groupBox_maxDistance">
     <property name="title">
      <string>Maximum distance</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="m_checkBox_maxDistance">
        <property name="text">
         <string>Enable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="m_doubleSpinBox_maxDistance">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> nm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_groupBox_ordenationTable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Ordenation table</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DistanceInformationOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DistanceInformationOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
